zokrates_cli/examples/imports/import_github.code

```diff
@@ -1,6 +0,0 @@
-// See also mock URLs in zokrates_github_resolver
```

Comment:
This example was removed without a replacement? Why is this the case? We still support imports from github.

Reply:
In theory because it requires network access and it's bad. In practice because it targets master, on which code files are still `.code`, not `.zok`, so we get a 404.

Reply:
Shall we create an issue/separate PR to test it back after we brought the extension changes to master? Don't think requiring network access during testing is horrible.

Reply:
Addressed in #481